parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -376,6 +381,15 @@ pub type AssetFeeAsExistentialDepositMultiplierFeeCharger = AssetFeeAsExistentia
 	TrustBackedAssetsInstance,
 >;
 
+/// Cases where a remote origin is accepted as trusted Teleporter:
+/// - teleportation of KSM from the parent Relay Chain and sibling system parachains; and
+/// - teleportation of sibling parachain's assets (as ForeignCreators).
+pub type TrustedTeleporters = (
+	NativeAsset,
+	IsForeignConcreteAsset<FromSiblingParachain<parachain_info::Pallet<Runtime>>>,
```

Comment:
This is a bit silly, couldn't we just have `IsSiblingParachainAsset` and call it a day? The code for `IsForeignConcreteAsset` mostly delegates to the `IsForeign` type parameter anyway.